### DISCUSSION TITLE
Citation Query

### DISCUSSION THREAD

**Discussion by dewanhaque1** [bastianvandenbout/LISEM#17] [Q&A]
Please share the original article on the safety factor equation of landslide slope stability.

**Comment by bastianvandenbout** [comment]
Hi, 

Apologies for the late reply here,
This one can be used:

Van den Bout, B., Lombardo, L., Chiyang, M., van Westen, C., & Jetten, V. (2021). Physically-based catchment-scale prediction of slope failure volume and geometry. Engineering geology, 284, 105942.

Kind regards,
Bastian
